tests/integration/olp-cpp-sdk-dataservice-read/VersionedLayerClientTest.cpp

```diff
@@ -247,6 +248,41 @@ class DataserviceReadVersionedLayerClientTest : public ::testing::Test {
                                    olp::http::HttpStatusCode::OK),
                                HTTP_RESPONSE_BLOB_DATA_PREFETCH_6));
 
+    ON_CALL(*network_mock_, Send(IsGetRequest(URL_BLOB_DATA_269), _, _, _, _))
+        .WillByDefault(
+            ReturnHttpResponse(olp::http::NetworkResponse().WithStatus(200),
+                               HTTP_RESPONSE_BLOB_DATA_269));
+
+    ON_CALL(*network_mock_,
+            Send(IsGetRequest(URL_QUERY_PARTITION_269), _, _, _, _))
+        .WillByDefault(
+            ReturnHttpResponse(olp::http::NetworkResponse().WithStatus(200),
+                               HTTP_RESPONSE_PARTITION_269));
+
+    ON_CALL(*network_mock_,
+            Send(IsGetRequest(URL_QUERY_PARTITION_269_V2), _, _, _, _))
+        .WillByDefault(
+            ReturnHttpResponse(olp::http::NetworkResponse().WithStatus(200),
+                               HTTP_RESPONSE_PARTITION_269_V2));
+
+    ON_CALL(*network_mock_,
+            Send(IsGetRequest(URL_BLOB_DATA_269_V2), _, _, _, _))
+        .WillByDefault(
+            ReturnHttpResponse(olp::http::NetworkResponse().WithStatus(200),
+                               HTTP_RESPONSE_BLOB_DATA_269_V2));
+
+    ON_CALL(*network_mock_,
+            Send(IsGetRequest(URL_QUERY_PARTITION_269_V10), _, _, _, _))
+        .WillByDefault(
+            ReturnHttpResponse(olp::http::NetworkResponse().WithStatus(400),
```

Comment:
```suggestion
            ReturnHttpResponse(olp::http::NetworkResponse().WithStatus(olp::http::HttpStatusCode::BAD_REQUEST),
```
Also in other places.